.vscode/settings.json

```diff
@@ -1,3 +1,2 @@
 {
-    "restructuredtext.confPath": "${workspaceFolder}\\src"
```

Comment:
maybe we should just gitignore this file

Reply:
Done. Thanks Robin.